src/lang/modifyAst.ts

```diff
@@ -339,6 +340,16 @@ export function sketchOnExtrudedFace(
   node: Node<Program>,
   sketchPathToNode: PathToNode,
   extrudePathToNode: PathToNode,
+  addTagForSketchOnFace: (
+    tagInfo: AddTagInfo,
+    expressionName: string,
+    edgeCutMeta: EdgeCutInfo | null
+  ) =>
+    | {
+        modifiedAst: Node<Program>
+        tag: string
+      }
+    | Error,
```

Comment:
Just checking, does this also break the circular dep? Because it's importing _types_ not concrete implementations?


```suggestion
  import { type addTagForSketchOnFace: AddTagForSketchOnFaceFn } from '@src/lang/std/sketch'
  
  // all the code...
  
  addTagForSketchOnFace: AddTagForSketchOnFaceFn,
```

Reply:
(I'm abusing the suggestion block so don't directly commit that)